mava/wrappers/system_trainer_statistics.py

```diff
@@ -149,6 +149,112 @@ def _compute_statistics(self, data: Dict[str, Dict[str, float]]) -> None:
                 self._network_loggers[network].write(network_running_statistics)
 
 
+class ScaledTrainerStatisticsBase(TrainerWrapperBase):
```

Comment:
I'm guessing these classes will also become the default once we convert all the systems? I.e. we will replace the `TrainerStatisticsBase` and related classes with these.

Reply:
Yes, exactly. Hopefully, this becomes the default trainer wrapper.